manager/metrics/collector.go

```diff
@@ -11,15 +11,17 @@ import (
 )
 
 var (
-	ns          = metrics.NewNamespace("swarm", "manager", nil)
-	nodesMetric metrics.LabeledGauge
+	ns            = metrics.NewNamespace("swarm", "manager", nil)
```

Comment:
This can be directly moved into `manager.go` - go-metrics best practice is to have metrics live next to the code that's affecting them